### PR TITLE
Add lints for idiomatic balloon alert usage

### PR DESCRIPTION
**Pull request by Mothblocks**
Adds lints for `balloon_alert(span_xxx(...))` (which is always wrong), and balloon alert where the first letter is a capital (which is usually wrong). Fixes everything that failed them. As a reminder, abbreviations like "AI" and "GPS" shouldn't be capitalized in a balloon alert.

In cases where this is intentional for flavor (there was one case), you can `UNLINT` like so:

```dm
balloon_alert(user, UNLINT("OMGWTFBBQ!!!"))
```

:cl:
qol: Several balloon alerts have been changed to be more consistent.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
would UNLINT work for this? I thought that existed for just spacemandmm?

**Comment by Mothblocks** [comment]
It's a magic trick.

The regex checks for `balloon_alert\(.*?, ?"[A-Z]`, AKA:

`balloon_alert(thing, "A`

And it's very specific about that, because we don't have dm-lua yet to perform actual intelligent parsing.

Meaning that if you do:

`balloon_alert(thing, UNLINT("A"))`

...then it doesn't find `, "A`, it finds `, U`, which is not what it's parsing.

File: code/modules/mod/modules/modules_maint.dm [hunk 1]
```diff
@@ -231,7 +231,7 @@
 	if(prob(min(num_sheets_dispensed * 2, 30)))
 		if(crisp_paper in mod.wearer.held_items)
 			mod.wearer.dropItemToGround(crisp_paper, force = TRUE)
-		crisp_paper.balloon_alert(mod.wearer, "PC LOAD LETTER!")
+		crisp_paper.balloon_alert(mod.wearer, UNLINT("PC LOAD LETTER!"))
```
Comment (Mothblocks) [inline code/modules/mod/modules/modules_maint.dm:234]: @LemonInTheDark

Comment (LemonInTheDark) [reply code/modules/mod/modules/modules_maint.dm:234]: damn chad

File: tools/ci/check_grep.sh [hunk 2]
```diff
@@ -274,12 +274,27 @@ if $grep '^/[\w/]\S+\(.*(var/|, ?var/.*).*\)' $code_files; then
...
 part "balloon_alert sanity"
 if $grep 'balloon_alert\(".*"\)' $code_files; then
 	echo
 	echo -e "${RED}ERROR: Found a balloon alert with improper arguments.${NC}"
 	st=1
 fi;
+
+if $grep 'balloon_alert(.*span_)' $code_files; then
+	echo
+	echo 'e "${RED}ERROR: Balloon alerts should never contain spans.${NC}"'
```
Comment (ZephyrTFA) [inline tools/ci/check_grep.sh:287]:
```suggestion
	echo -e "${RED}ERROR: Balloon alerts should never contain spans.${NC}"'
```

Comment (ZephyrTFA) [inline tools/ci/check_grep.sh:287]:
```suggestion
	echo -e "${RED}ERROR: Balloon alerts should never contain spans.${NC}"
```

**Pull request merged**